Portugues.json

```
@@ -0,0 +1,159 @@
+{
+    "status": {
+        "terminated": "\nAplicacao Finalizada.\nFeche a janela para sair do programa.",
+        "goes_online": "{channel} esta ONLINE, Mudando...",
+        "goes_offline": "{channel} ficou OFFLINE, Mudando...",
+        "claimed_drop": "Coletou o drop: {drop}",
+        "claimed_points": "Coletou o bonus: {points}",
+        "earned_points": "Ganhou este valor de pontos: {points}, total: {balance}",
+        "no_channel": "Sem canal disponivel para assisitir. Esperando por um canal ONLINE...",
+        "no_campaign": "Sem campanhas ativas no momento para minerar. Esperando por uma campanha ativa..."
+    },
+    "login": {
+		"chrome": {

```

And here is two tabs instead of 8 spaces like it should've been. This is probably the last oustanding change in this PR.

```suggestion
        "chrome": {
```